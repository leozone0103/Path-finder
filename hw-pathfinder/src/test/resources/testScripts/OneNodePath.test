# Test findPath from a single node to itself
CreateGraph graph
AddNode graph A
FindPath graph A A