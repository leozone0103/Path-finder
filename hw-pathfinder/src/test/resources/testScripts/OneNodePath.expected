# Test findPath from a single node to itself
created graph graph
added node A to graph
path from A to A:
total cost: 0.000